infra/bootstrapping/check-readme.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+import os
+
+
+argParser = argparse.ArgumentParser()
+argParser.add_argument("working_directory", help="The working directory for this script.")
+argParser.add_argument("sample_path", help="The absolute path to the sample directory.")
+args = argParser.parse_args()
+working_directory = args.working_directory.replace("\\", "/")
+sample_path = args.sample_path.replace("\\", "/")
```

Comment:
```suggestion
import argparse
from pathlib import Path



argParser = argparse.ArgumentParser()
argParser.add_argument("sample_path", help="The absolute path to the sample directory.", type=Path)
args = argParser.parse_args()
working_directory = Path(__file__).parent.parent
sample_path = args.sample_path
```

Reply:
Using `pathlib` lets you avoid some the path normalization.

It looks like the intent is that `working_directory` is meant to always point to the root of the repository. You can use the dunder variable `__file__` to just hardcode that.